keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ArtifactSummary.kt

```diff
@@ -2,12 +2,14 @@ package com.netflix.spinnaker.keel.core.api
 
 import com.fasterxml.jackson.annotation.JsonIgnore
 import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
 import com.netflix.spinnaker.keel.api.artifacts.ArtifactType
 import java.time.Instant
 
 /**
  * Summarized data about a specific artifact, mostly for use by the UI.
  */
+@JsonPropertyOrder(value = ["name", "type", "versions"])
```

Comment:
@erikmunson This now ensures these properties show up first, in the specified order, in artifact summaries.